src/components/Quantity/Quantity.jsx

```diff
@@ -1,11 +1,26 @@
 import { useState } from "react";
+import React from "react";
 import Input from "../Input/Input";
 import Button from "../Button/Button";
 
 import "./Quantity.scss";
 
+export const AuthContext = React.createContext({});

```

Comment:
I recommend you move this one to its own file and change the name of this context:

```jsx
// cart-context.js

export const cartDefaultValue = { // we can use it on App to set the default state.
  quantity: 0,
}

export const AuthContext = React.createContext(cartDefaultValue);
```